### ISSUE TITLE
upload file name extension "." to "-" 

### ISSUE THREAD

**Issue by capitalfuse** [joni2back/angular-filemanager#278]
Dear Sir or Madam,
I appreciate for the angular-filemanager application developers.
I don't understand why upload file name extension changed to "-" from "." when uploaded a file from other PC.
Example) changed "music.mp3" to "music-mp3".
So, I changed the following code, inactive "private function normalizeName($name)"
**bridges/php-local/LocalBridge/FileManagerApi.php**
```
    private function uploadAction($path, $files)
    {
        $path = $this->canonicalizePath($this->basePath . $path);

        foreach ($_FILES as $file) {
            $uploaded = move_uploaded_file(
                $file['tmp_name'],
                $path . DIRECTORY_SEPARATOR . $file['name'] --->changed here!
            );
            if ($uploaded === false) {
                return false;
            }
        }

        return true;
    }
```
Fixed by above method, but am I right???

**Comment by durasj** [comment]
Hello @capitalfuse,

yes, you are right! Dot in the ext. shouldn't be replaced. But the solution is not to simply disable the normalization but instead, I applied the [normalization to the name of the file only](https://github.com/durasj/angular-filemanager/commit/0366ba82ad9bea4d2551cbb696b36c6b42dcbeb9). Thanks for reporting that!

Opened [PR with the fix](https://github.com/joni2back/angular-filemanager/pull/280) and it's also already available in the [current bundle](https://github.com/durasj/angular-filemanager/raw/bundle-php-local/bundle/master-2017-06-03.zip).

**Comment by joni2back** [comment]
fixed in #280
